test/Control.t.sol

```diff
@@ -183,18 +190,19 @@ contract Control is Test {
         assertEq(blb.balanceOf(address(blueberryStaking)),(1e19 + (1e19 * 4) + (1e23 * 4)));
     }
 
-    // Test changing the stable token address
-    function testChangeStable() public {
+    // Test changing the uniswap pool
+    function testChangeUniswapPool() public {
         vm.startPrank(owner);
-
-        // Deploy a new stable coin 
-        MockToken token = new MockToken(9);
         
```

Comment:
Would be good to also add a tests that expects to revert when setting a stable token that is not in the Uniswap pair